src/reistijden_v1/tests.py

```diff
@@ -0,0 +1,383 @@
+                        <camera id="111">
+                            <coordinates latitude="52.111111" longitude="4.111111" />
+                            <laneNumber>1</laneNumber>
+                            <status>on</status>
+                            <viewDirection>111</viewDirection>
+                        </camera>
+                    </lane>
+                </location>
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+        <siteMeasurements>
+            <measurementSiteReference id="22" version="1.0">
+                <measurementSiteType>location</measurementSiteType>
+                <location>
+                    <lane specificLane="lane1">
+                        <camera id="222">
+                            <coordinates latitude="52.222222" longitude="4.222222" />
+                            <laneNumber>1</laneNumber>
+                            <status>on</status>
+                            <viewDirection>222</viewDirection>
+                        </camera>
+                    </lane>
+                </location>
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+        <siteMeasurements>
+            <measurementSiteReference id="33" version="1.0">
+                <measurementSiteType>location</measurementSiteType>
+                <location>
+                    <lane specificLane="lane1">
+                        <camera id="333">
+                            <coordinates latitude="52.333333" longitude="4.333333" />
+                            <laneNumber>1</laneNumber>
+                            <status>on</status>
+                            <viewDirection>333</viewDirection>
+                        </camera>
+                    </lane>
+                </location>
+            </measurementSiteReference>
+            <trafficFlowData>
+                <measuredFlow specificLane="lane1">
+                    <vehicleFlow>6</vehicleFlow>
+                    <numberOfInputValuesUsed>
+                        <category count="6" type="Auto" />
+                    </numberOfInputValuesUsed>
+                </measuredFlow>
+            </trafficFlowData>
+        </siteMeasurements>
+    </payloadPublication>
+</amsterdamTravelTimes>
+"""
```

Comment:
Ik zou dit (maar dat is puur persoonlijke voorkeur) naar een aparte file halen die naast de testfile leeft. Iets van `test_xml.py` of iets dergelijks.